src/gui/render.cpp

```diff
@@ -609,8 +609,8 @@ static bool RENDER_GetShader(std::string& shader_path) {
 	else if (shader_path == "tv3x")        buf << tv3x_glsl;
 	else if (shader_path == "sharp")       buf << sharp_glsl;
 
-	if (!buf.str().empty()) {
-		std::string s = buf.str();
+	std::string s = buf.str();
```

Comment:
Is `s` a reference or copy?  (Just noticed that further down we update `s`, so probably need to be reference).